<?xml version="1.0"?>
<launch>
    <node pkg="rosserial_python" type="serial_node.py" name="serial_node">
        <!-- <param name="port" value="/dev/arduinoPWM"/> -->
        <param name="port" value="/dev/ttyACM0"/>
        <param name ="baud" value="250000" />
        <remap from="/lli/ctrl_request" to="/SVEA0/lli/ctrl_request" />
        <remap from="/lli/ctrl_actuated" to="/SVEA0/lli/ctrl_actuated" />
    </node>
    
    <node pkg="svea" type="lli_test.py" name="lli_test" output="screen">
    </node>

</launch>
